packages/lib/services/e2ee/EncryptionService.ts

```diff
@@ -72,10 +75,11 @@ export default class EncryptionService {
 	//
 	// So making the block 10 times smaller make it 100 times faster! So for now using 5KB. This can be
 	// changed easily since the chunk size is incorporated into the encrypted data.
-	private chunkSize_ = 5000;
+	public chunkSize_ = 65536;
 	private decryptedMasterKeys_: Record<string, DecryptedMasterKey> = {};
-	public defaultEncryptionMethod_ = EncryptionMethod.SJCL1a; // public because used in tests
-	private defaultMasterKeyEncryptionMethod_ = EncryptionMethod.SJCL4;
+	public defaultEncryptionMethod_ = EncryptionMethod.StringV1; // public because used in tests
+	public defaultFileEncryptionMethod_ = EncryptionMethod.FileV1; // public because used in tests
```

Comment:
I just realise we cannot change the defaults right away, as if there's any bug that could lead to data loss during sync. Instead please do the following:

- Set the above values back to their defaults. INCLUDING chunk size.
- Create a new "Use beta encryption method" feature flag (search for FeatureFlag in the codebase to see how it works)
- Set to "false" by default

When it's "true", set the above values to your new encryption methods.

Regarding the chunk size, if it needs to be different for your new method, it means it should be a property of the encryption method. Basically it should no longer be defined here.

Reply:
I added `featureFlag` for enabling/disabling the new encryption method. (commit bf0f12628e6522958e500ba93cfc678894ad1ca8)
For the chunk size, I added the chunk size map to the `chunkSize()` method. (commit 61dd12d0033eecd5bbafdb2557a3b90726d16004)